pkg/kotscli/kotscli.go

```diff
@@ -25,7 +26,8 @@ type InstallOptions struct {
 }
 
 func Install(opts InstallOptions, msg *spinner.MessageWriter) error {
-	kotsBinPath, err := goods.MaterializeInternalBinary("kubectl-kots")
+	materializer := goods.NewMaterializer(defaults.EmbeddedClusterHomeDirectory())
```

Comment:
shouldn't this use the runtimeconfig?

Reply:
this pr only attempts to consolidate the directories. not allow them to be overwritten. i will do that in a follow up

Reply:
It now allows for overwriting